### ISSUE TITLE
in app upload process enhancement

### ISSUE THREAD

**Issue by loicginoux** [loicginoux/joanne_project#245]
this is a discussion to see if we could improve the way we upload in the app...

**Comment by loicginoux** [comment]
I agree we should have a way to process the photo to make it nicer, having filters, resize, rotate, etc... maybe the file.io is actually more and more needed...

**Comment by jdaudier** [comment]
A friend also mentioned that maybe we can make a mobile version of our site with big buttons for easy upload from a phone.

**Comment by loicginoux** [comment]
yes that's another option, there are different discussions about how to make your web app mobile and which solution to implement. I read this recently about this question which is quite interesting.
http://sixrevisions.com/mobile/methods-mobile-websites/
